src/signals/incident-management/components/FilterForm/components/Notification.tsx

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2022 Gemeente Amsterdam
+import type { FC, MutableRefObject } from 'react'
+import { useEffect } from 'react'
+
+import { Alert } from '@amsterdam/asc-ui'
+import { StyledNotification } from '../styled'
+
+export type Props = {
+  reference: MutableRefObject<HTMLDivElement | null>
```

Comment:
is it possible to make it 
```
reference?: MutableRefObject<HTMLDivElement>
```
or will a reference always be mutableObject with null inside?

Reply:
The second indeed. reference.current is either `null` or a div element